docs/user_guide/header-links.rst

```diff
@@ -188,6 +200,54 @@ For example:
 
    Use ``.svg`` images for a higher-resolution output that behaves similarly across screen sizes.
 
+.. _svg-image-icons:
+
+SVG image icons
+~~~~~~~~~~~~~~~
+
+In order to make use of the full feature set of ``.svg`` images provided by HTML you will need
+to set up the ``.svg`` to be used as a FontAwesome type icon. This is a fairly straightforward process:
+
+#. Copy the contents of ``docs/_static/custom-icon.js`` into an appropriate directory of your documentation
+   source (typically ``source/js``) and rename the file however you like.
+
+   .. literalinclude::  ../_static/custom-icon.js
+    :language: javascript
+    :emphasize-lines: 7, 13
+    :linenos:
```

Comment:
Would it be possible to copy/paste only the meaningful content in a standard code-block ? We cannot guarantee this file will remain as it is forever and I would prefer the documentation not to depend on our internal cuisine.